src/components/Favorites/Favorite.component.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { List } from 'immutable';
+import propTypes from 'prop-types';
+
+import LinkToSongContainer from '../Files/LinkToSong';
+
+const FavoriteComponent = ({ favorites }) => (
+  <div className="content-container">
+    <h1 id="your-personal-library" className="title">
```

Comment:
Curious about the reason behind the existence of this id. I don't see where it is used.

Reply:
Since we only have the route title in screen once, i thought it was a good idea to target its styles with id. It has a className 'title', yes. But that className is used to target font-family and font-weight. 😄

Reply:
Let's avoid using ids for styling purposes. Having ids will make CSS rules more complicated. Also, we are removing the current css implementation anyway.